mitodl/odl-video-service
# Store x509 certificate/key as environment variables

#### What are the relevant tickets?
Fixes #142 
Dependent on https://github.com/mitodl/mit-moira/pull/3

#### What's this PR do?
Gets the contents of an x509 certificate and private key as environmental variables, and saves them to files on startup.  Uncomments `ui.utils.get_moira_client` functionality.

#### How should this be manually tested?
- In your .env file, set `MIT_WS_CERTIFICATE` to the contents of your x509 certificate file and `MIT_WS_PRIVATE_KEY` to the contents of your x509 private key (I can provide a sample).
- `docker-compose build; docker-compose run web python manage.py shell`
- ```
  from ui.models import MoiraList
  odl_list = MoiraList(name='odl-engineering')
  odl_list.members()
  ```
- The above should return a list of all users in the odl-engineering list, including Anna's email, with no duplicates (assuming https://github.com/mitodl/mit-moira/pull/3 has been merged)
